ee/codegen_integration/fixtures/simple_map_node/code/nodes/map_node/nodes/search_node.py

```diff
@@ -1,7 +1,7 @@
 from vellum import SearchFiltersRequest, SearchRequestOptionsRequest, SearchResultMergingRequest, SearchWeightsRequest
 from vellum.workflows.nodes.displayable import SearchNode as BaseSearchNode
 
-from ....inputs import Inputs
+from ....nodes.map_node.inputs import Inputs
```

Comment:
This file is inside the subworkflow. We should be using the `Inputs` from the subworkflow, NOT the outer workflow.